src/Umbraco.Web/Editors/ContentController.cs

```diff
@@ -179,18 +179,22 @@ public HttpResponseMessage GetNiceUrl(int id)
         /// <returns></returns>        
         [FilterAllowedOutgoingContent(typeof(IEnumerable<ContentItemBasic<ContentPropertyBasic, IContent>>), "Items")]
         public PagedResult<ContentItemBasic<ContentPropertyBasic, IContent>> GetChildren(
-            int id, 
-            int pageNumber = 0,  //TODO: This should be '1' as it's not the index
-            int pageSize = 0, 
-            string orderBy = "SortOrder", 
-            Direction orderDirection = Direction.Ascending, 
-            string filter = "")
+                int id,
+                int pageNumber = 0,  //TODO: This should be '1' as it's not the index
+                int pageSize = 0,
+                string orderBy = "SortOrder",
+                Direction orderDirection = Direction.Ascending,
+                int? orderBySystemField = 2,

```

Comment:
The value is posted as a number, and for some reason it wouldn't recognize it properly when the signature received a bool.
I set it to 2 by default to know if it was even posted, then I convert to bool depending on the value.
It was just a quick fix to make it work. Ideally bool would work out of the box.